src/app/Settings/FeatureLevels.tsx

```diff
@@ -99,4 +99,6 @@ export const FeatureLevels: UserSetting = {
   title: 'Feature Level',
   description: 'Control which graphical features appear in the application.',
   content: Component,
+  category: 'General',
+  orderInGroup: 1,
```

Comment:
I think this should go under `Advanced` actually.

Reply:
If this is moved to `Advanced` then the remaining two items under `General` are both seldom-used options relating to WebSockets, so perhaps `General` should be renamed to `WebSockets` or maybe `Connectivity`.

Reply:
I removed `General` for now until we have some general settings later :D